### PR TITLE
refactor(settings): Replace security annotations with respective attributes

### PR DESCRIPTION
**Pull request by provokateurin**
## Summary

Follow-up to https://github.com/nextcloud/server/pull/46606

## Checklist

- Code is [properly formatted](https://docs.nextcloud.com/server/latest/developer_manual/digging_deeper/continuous_integration.html#linting)
- [Sign-off message](https://github.com/src-d/guide/blob/master/developer-community/fix-DCO.md) is added to all commits
- [ ] Tests ([unit](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#unit-tests), [integration](https://docs.nextcloud.com/server/latest/developer_manual/app_development/tutorial.html#integration-tests), api and/or acceptance) are included
- [ ] Screenshots before/after for front-end changes
- [ ] Documentation ([manuals](https://github.com/nextcloud/documentation/) or wiki) has been updated or is not required
- [ ] [Backports requested](https://github.com/nextcloud/backportbot/#usage) where applicable (ex: critical bugfixes)

### CODE REVIEW

File: apps/settings/lib/Controller/AdminSettingsController.php [hunk 1]
```diff
@@ -46,12 +48,12 @@ public function __construct(
 	}
 
 	/**
-	 * @NoCSRFRequired
-	 * @NoAdminRequired
 	 * @NoSubAdminRequired
```
Comment (come-nc) [inline apps/settings/lib/Controller/AdminSettingsController.php:51]: No attribute for this one?

Comment (provokateurin) [reply apps/settings/lib/Controller/AdminSettingsController.php:51]: They were already present: https://github.com/nextcloud/server/blob/f012c996ee378fac2238d9ca7ada378d20e03723/apps/settings/lib/Controller/AdminSettingsController.php#L55-L56

Comment (come-nc) [reply apps/settings/lib/Controller/AdminSettingsController.php:51]: I meant for `NoSubAdminRequired`

Comment (provokateurin) [reply apps/settings/lib/Controller/AdminSettingsController.php:51]: Oh I misread the question. No there is no corresponding attribute for it yet.

**Review by come-nc — APPROVED** [review]

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**